web/thesauruses/c/17/data_types.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
You're missing a LOT of the IDs from this file still. Every time an ID is deleted, it creates an "unknown" in the software. If there's no type for it, you should make it `"not-implemented": true` instead. (though I think most of the types do exist in C and are the same as C++.)

![image](https://user-images.githubusercontent.com/2601974/197648093-6e897fca-34ae-4ef1-86b0-9d1c827867a8.png)